src/main/resources/application.properties

```diff
@@ -1,10 +1,13 @@
-spring.application.name=demo
+spring.application.name=bookstoreBakersButBetter
 server.port=8080
```

Comment:
needs 2 be changed to 8081 when lauren pushes

Reply:
okay! Ill just put that change in my code for just incase